src/main/java/com/qa/trainer/service/TrainerServiceImpl.java

```diff
@@ -1,15 +1,54 @@
 package com.qa.trainer.service;
 
+import java.util.List;
+
 import org.springframework.stereotype.Service;
 
+import com.qa.trainer.entities.Constants;
+import com.qa.trainer.entities.CreateAccount;
+import com.qa.trainer.entities.Trainer;
+import com.qa.trainer.entities.UpdateAccount;
+
 @Service
 public class TrainerServiceImpl implements TrainerService {
-
+	
+	
+	public String checkDuplicates(CreateAccount accountC, List<Trainer> trainers) {
+		Trainer matchingTrains = new Trainer();
```

Comment:
```suggestion
		Trainer trainer = new Trainer();
```

Reply:
changed to matchingTrainer as trainer is used inside the stream